tests/fixtures/modules/invalid-with-statement-in-module.result.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
This test is failing in Travis because it expects to throw an error. Probably need to move this out into `ecma-features-mix` to avoid that.